bin/satellite-clone.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import subprocess
+import sys
+
+def yes_or_no(question):
+    while "the answer is invalid":
+        reply = str(raw_input(question+' (y/n): ')).lower().strip()
+        if reply[0] == 'y':
+            return True
+        if reply[0] == 'n':
+            exit(1)
+
+def main():
+    parser = argparse.ArgumentParser(usage="Usage: satellite-clone [options]\n" \
+                                           "Example: satellite-clone --start-at-task=\"Clean yum info\"")
+    parser.add_argument("-v", "--verbose",
+                      action="store_true",
+                      dest="verbose",
+                      default=False,
+                      help="verbose output")
+    parser.add_argument("--start-at-task",
+                      action="store", # optional because action defaults to "store"
+                      dest="start_at_task",
+                      help="Start at a specific task",)
+    args = parser.parse_args()
+
+
+    DEFAULT_PRODUCTION_INSTALL_PATH = "/usr/share/satellite-clone"
+    DEFAULT_PLAYBOOK_FILE = os.path.join(DEFAULT_PRODUCTION_INSTALL_PATH, "satellite-clone-playbook.yml")
+
+    yes_or_no("This will initiate satellite-clone. Do you want to proceed?")
```

Comment:
Would it make sense to have a ```--yes``` argument so it's no longer interactive?

Reply:
I thought about this, but satellite-clone is a pretty deliberate command to run and will heavily modify a system and overwrite data. A user also only has to run it once (or maybe a few) times. Even having `satellite-clone -y` in the history or as a misunderstood option makes me nervous about wiping someone's Satellite instance, not sure if I'm being overly paranoid, but the risk/reward doesn't seem worth it.

Reply:
That's fair.